src/core/middleware.py

```diff
@@ -77,7 +79,7 @@ def __call__(self, request):
         if response.status_code > 399:
             directives = error_directives
 
-        if not response.get("Cache-Control"):
+        if not response.get("Cache-Control", None):
```

Comment:
covered in the 'ping' test?

Reply:
I think the problem was that 'Cache-Control' was there, but it was 'None'. This wouldn't fix it.

Reply:
reverted, can't replicate original problem